tools/dhs/include/dhs_patch/dhs_patch_compat.h

```diff
@@ -131,4 +164,4 @@ dhs_a9_compat_s dhs_a9_compat_49 = {
 
 #endif /*DHS_PATCH_COMPAT_DATA*/
 
-#endif /*__DHS_PATCH_COMPAT_H*/
+#endif /*DHS_PATCH_COMPAT_H*/
```

Comment:
I agree with all changes except for these.
Any specific reason why you want to move the file declarations to compat?